src/pages/RemoveLiquidity/index.tsx

```diff
@@ -510,152 +503,54 @@ export default function RemoveLiquidity({
-                      <RowBetween style={{ justifyContent: 'flex-end' }}>
-                        {oneCurrencyIsAVAX ? (
-                          <StyledInternalLink
-                            to={`/remove/${currencyA === CETH ? WETH[chainId].address : currencyIdA}/${
-                              currencyB === CETH ? WETH[chainId].address : currencyIdB
-                            }`}
-                          >
-                            {t('removeLiquidity.receiveWmatic')}
-                          </StyledInternalLink>
-                        ) : oneCurrencyIsWETH ? (
-                          <StyledInternalLink
-                            to={`/remove/${
-                              currencyA && currencyEquals(currencyA, WETH[chainId]) ? 'AVAX' : currencyIdA
-                            }/${currencyB && currencyEquals(currencyB, WETH[chainId]) ? 'AVAX' : currencyIdB}`}
-                          >
-                            {t('removeLiquidity.receiveMatic')}
-                          </StyledInternalLink>
-                        ) : null}
-                      </RowBetween>
-                    ) : null}
-                  </AutoColumn>
-                </LightCard>
-              </>
-            )}
+            <CurrencyInputPanel
+              value={formattedAmounts[Field.LIQUIDITY]}
+              onUserInput={onLiquidityInput}
+              onClickBalanceButton={value =>
+                onUserInput(Field.LIQUIDITY_PERCENT, value === BalanceButtonValueEnum.MAX ? '100' : '50')
+              }
+              disableHalfButton={atHalfAmount}
+              disableMaxButton={atMaxAmount}
+              disableCurrencySelect
+              currency={pair?.liquidityToken}
+              pair={pair}
+              id="liquidity-amount"
+            />
+            <ColumnCenter>
+              <ArrowDown size="16" color={theme.text2} />
+            </ColumnCenter>
+            <CurrencyInputPanel
+              hideBalance={true}
+              value={formattedAmounts[Field.CURRENCY_A]}
+              onUserInput={onCurrencyAInput}
+              onClickBalanceButton={value =>
+                onUserInput(Field.LIQUIDITY_PERCENT, value === BalanceButtonValueEnum.MAX ? '100' : '50')
+              }
+              disableHalfButton={atHalfAmount}
+              disableMaxButton={atMaxAmount}
+              currency={currencyA}
+              label={t('removeLiquidity.output')}
+              onCurrencySelect={handleSelectCurrencyA}
+              id="remove-liquidity-tokena"
+            />
+            <ColumnCenter>
+              <Plus size="16" color={theme.text2} />
+            </ColumnCenter>
+            <CurrencyInputPanel
+              hideBalance={true}
+              value={formattedAmounts[Field.CURRENCY_B]}
+              onUserInput={onCurrencyBInput}
+              onClickBalanceButton={value =>
+                onUserInput(Field.LIQUIDITY_PERCENT, value === BalanceButtonValueEnum.MAX ? '100' : '50')
```

Comment:
Quick thought -- would it be easier if this enum was set up link this instead:

```
const enum BalanceButtonValueEnum {
    MAX = new Percent(100);
    HALF = new Percent(50);
}
```

Reply:
ended up not using this